### DISCUSSION TITLE
build failed: Unable to sync pages directory. Please try again later.

### DISCUSSION THREAD

**Discussion by 9fogel** [community/community#35690] [Other Feature Feedback, Questions, & Ideas]
Hello, I merged a pull request from development branch to gh-pages branch and received the following **email with error notice**:

_The page build failed for the `gh-pages` branch with the following error:
Unable to sync pages directory. Please try again later._

When I open deploy link I see no recent changes (changes from latest commits were not applied).
When I try to open a new pull request, it's said that _There isn’t anything to compare._

I'm not sure what to do in order to fix the error and to have my deploy updated with latest changes.
This is quite urgent as the deploy is needed for my education and deadline to present it is today's evening

**Comment by fullstop125** [comment]
hello @9fogel can you please link the repository you wish to have it looked at?

**Reply by 9fogel** [reply]
https://github.com/rolling-scopes-school/9fogel-JSFE2022Q3/tree/gh-pages

here is the link to my school repo, where I have this issue with deploy

**Comment by fullstop125** [comment]
@9fogel  could you check the address link again ? This is what i get from your link ...
one of the causes could be maybe it's a private repository or a private organization repository


![image](https://user-images.githubusercontent.com/35464104/194907884-f9fb1713-6a6c-42f2-92fc-048fa53ea275.png)

**Reply by 9fogel** [reply]
yeah, it's private. However looks like I fixed the problem - I made a small change and new commit helped to deploy successfully. Anyway thanks for assistance!

**Reply by fullstop125** [reply]
sure ...welcome 💯
